### ISSUE TITLE
so self:update triggers a PHP Fatal on Linux

### ISSUE THREAD

**Issue by dpellenwood** [moderntribe/square1-global-docker#45]
**Describe the bug**
Ran `so self:update` to grab the latest version. Update appears to have worked fine, but I got a PHP fatal afterwards.  Here's the console output:

```
> so self:update
Updating SquareOne to 3.0.4...
Successfully updated to 3.0.4.
PHP Fatal error:  Uncaught ErrorException: include(phar:///home/dpe/.config/squareone/bin/so/vendor/composer/../../app/Exceptions/Handler.php): failed to open stream: phar error: internal corruption of phar "/home/dpe/.config/squareone/bin/so" (actual filesize mismatch on file "app/Exceptions/Handler.php") in phar:///home/dpe/.config/squareone/bin/so/.box/vendor/composer/ClassLoader.php:444
Stack trace:
#0 phar:///home/dpe/.config/squareone/bin/so/.box/vendor/composer/ClassLoader.php(444): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError()
#1 phar:///home/dpe/.config/squareone/bin/so/.box/vendor/composer/ClassLoader.php(444): include()
#2 phar:///home/dpe/.config/squareone/bin/so/.box/vendor/composer/ClassLoader.php(322): Composer\Autoload\includeFile()
#3 [internal function]: Composer\Autoload\ClassLoader->loadClass()
#4 [internal function]: spl_autoload_call()
#5 phar:///home/dpe/.config/squareone/bin/so/vendor/illuminate/container/Container.php(809): ReflectionClass->__construct()
#6 phar:///home/dpe/.config/sq in phar:///home/dpe/.config/squareone/bin/so/.box/vendor/composer/ClassLoader.php on line 444
PHP Fatal error:  Uncaught ErrorException: include(phar:///home/dpe/.config/squareone/bin/so/vendor/composer/../symfony/error-handler/Error/FatalError.php): failed to open stream: phar error: internal corruption of phar "/home/dpe/.config/squareone/bin/so" (actual filesize mismatch on file "vendor/symfony/error-handler/Error/FatalError.php") in phar:///home/dpe/.config/squareone/bin/so/.box/vendor/composer/ClassLoader.php:444
Stack trace:
#0 phar:///home/dpe/.config/squareone/bin/so/.box/vendor/composer/ClassLoader.php(444): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError()
#1 phar:///home/dpe/.config/squareone/bin/so/.box/vendor/composer/ClassLoader.php(444): include()
#2 phar:///home/dpe/.config/squareone/bin/so/.box/vendor/composer/ClassLoader.php(322): Composer\Autoload\includeFile()
#3 [internal function]: Composer\Autoload\ClassLoader->loadClass()
#4 phar:///home/dpe/.config/squareone/bin/so/vendor/laravel-zero/foundation/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(143): spl_autoload_call()
#5  in phar:///home/dpe/.config/squareone/bin/so/.box/vendor/composer/ClassLoader.php on line 444
```

**System Configuration:**
 - OS: Ubuntu 20.04 LTS
 - PHP Version 7.4.3

**Comment by defunctl** [comment]
This was fixed in 3.0.2 https://github.com/moderntribe/square1-global-docker/releases/tag/3.0.2, but it's not fixed until you've updated and future updates have the fix (because the old phar runs the update).

If you do a `so self:update` now, it will still update your binary and you should no longer have this error.

Let me know!

**Comment by dpellenwood** [comment]
Worked great!
